lib/c-api/src/wasm_c_api/unstable/middlewares/mod.rs

```diff
@@ -0,0 +1,40 @@
+//! Unstable non-standard Wasmer-specific types to manipulate module
+//! middlewares.
+
+pub mod metering;
+
+use super::super::engine::wasm_config_t;
+use std::sync::Arc;
+use wasmer::ModuleMiddleware;
+
+#[cfg(all(feature = "middlewares", not(feature = "compiler")))]
+compile_error!("The `middlewares` feature requires the `compiler` feature to be turned on");
+
+/// Opaque representing any kind of middleware.
+///
+/// It's used by `wasm_config_push_middleware`. A specific middleware
```

Comment:
```suggestion
/// Used by `wasm_config_push_middleware`. A specific middleware
```

Or just not mention what it's used by?

Reply:
:+1: